lib/modules/ens/register.js

```diff
@@ -10,27 +10,27 @@ function registerSubDomain(ens, registrar, resolver, defaultAccount, subdomain,
   const toSend = ens.methods.setSubnodeOwner(rootNode, subnode, defaultAccount);
   let transaction;
 
-  secureSend(web3, toSend, {from: defaultAccount}, false)
+  secureSend(web3, toSend, {from: defaultAccount, gas: 700000}, false)
```

Comment:
Did you just put a big number to make sure it passes.
I guess this is only private chain?

Reply:
It is indeed only for dev and privatechain. Also, that's the rounded up estimation of what it costs. I didn't want to put estimate gas on all of those, especially since we kind of know how much it costs

Reply:
Can I be pedantic and ask you to make that into a constant, and comment on why it is the value that it is?

Later we'll forget why this decision was made and a comment is right there.